src/cljs/rems/administration/create_workflow.cljs

```diff
@@ -184,20 +190,40 @@
        :multi? true
        :on-change #(rf/dispatch [::set-handlers %])}]]))
 
+(defn- workflow-forms-field []
+  (let [all-forms @(rf/subscribe [::forms])
+        selected-form-ids (set (mapv :form/id (:forms @(rf/subscribe [::form]))))
+        id "workflow-forms"]
+    [:div.form-group
+     [:label {:for id} (text :t.administration/forms)]
+     [dropdown/dropdown
+      {:id id
+       :items all-forms
+       :item-key :form/id
+       :item-label :form/title ;; TODO render org like in create-catalogue-item
+       :item-selected? #(contains? selected-form-ids (:form/id %))
+       ;; TODO support selecting (& ordering!) multiple forms
+       :multi? false
```

Comment:
Why is this limited to one? I don't really see the point, could just use the multi-select capability for free.